tox.ini

```diff
@@ -41,6 +41,15 @@ commands =
     coverage combine
     coverage html
 
+[testenv:lint]
+skip_install = true
+deps =
+    black
+    flake8
+commands =
+    black --check --diff Lib/fontTools/pens Lib/fontTools/cu2qu/ufo.py Lib/fontTools/ufoLib/glifLib.py Lib/fontTools/varLib/interpolatable.py
+    flake8 Lib/fontTools/pens Lib/fontTools/cu2qu/ufo.py Lib/fontTools/ufoLib/glifLib.py Lib/fontTools/varLib/interpolatable.py
```

Comment:
Maybe these should be put into the config files like with mypy so we can more easily extend them as time goes on.

- https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-format
- https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-filename

Reply:
Actually, I would pin black and flake8 in dev-requirements.txt, because new versions can and will break CI when you're working on something else.